loki/ir.py

```diff
@@ -158,13 +158,17 @@ def args_frozen(self):
     def __repr__(self):
         raise NotImplementedError
 
-    def view(self):
+    def view(self, visualization = False, **kwargs_visualization):
```

Comment:
I think I'd separate the functionality into a new method, e.g. `visualize`? Given that this is likely to be used mostly interactively, it's shorter to write and muscle memory is easier/auto completion more reliable with a separate method than switching via optional arguments.

Reply:
has been separated into own function with explicit arguments, c72fe87359edd81a3f233e85478edf849f7b2c33